todo-mvc-tsx/src/main.ts

```diff
@@ -0,0 +1,23 @@
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import Route from '@dojo/routing/Route';
+import TodoApp from './widgets/TodoApp';
+import router from './routes';
+
+const root = document.querySelector('my-app') || undefined;
+
+const Projector = ProjectorMixin(TodoApp);
+const projector = new Projector();
+
+// TODO find a better place for this
```

Comment:
We should open an issue for this note to resolve it at some point and cross-reference (I know it's the same as in the other example)